Makefile.am

```diff
@@ -43,9 +43,7 @@ AM_CFLAGS = @STRICT_CFLAGS@
 # Make sure to run Gtk-doc tests and build the documentation when doing
```

Comment:
I think the commit message and pull request description are stale. When I run `./autogen.sh --enable-strict-flags --enable-gtk-doc --prefix=/usr && make && make check && make distcheck` on master at HEAD in eos-metrics, I hit the following error during the `make distcheck`. Also, docs/reference/eosmetrics-js is only made when I explicitly configure with `--enable-gir-doc`. That is why I don't think that GIR doc is enabled by default (without your changes in this pull request).

```
make[1]: se ingresa al directorio `/home/endless/endlessm/eos-metrics'
***
*** You must configure with --enable-gtk-doc and --enable-gir-doc to
*** run make dist or make distcheck.
***
make[1]: *** [dist-hook] Error 1
make[1]: se sale del directorio `/home/endless/endlessm/eos-metrics'
make: *** [distdir] Error 2
```